multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/madara/MadaraGenerator.kt

```diff
@@ -136,6 +136,7 @@ class MadaraGenerator : ThemeSourceGenerator {
         SingleLang("HM2D", "https://mangadistrict.com/hdoujin", "en", isNsfw = true, overrideVersionCode = 1),
         SingleLang("HouseMangas", "https://housemangas.com", "es"),
         SingleLang("Hreads", "https://hreads.net", "en", isNsfw = true, overrideVersionCode = 1),
+        SingleLang("Hunters Scans", "https://huntersscan.xyz", "pt-BR", isNsfw = true, overrideVersionCode = 1),
```

Comment:
```suggestion
        SingleLang("Hunters Scans", "https://huntersscan.xyz", "pt-BR", isNsfw = true),
```

Don't bump version unless you edited source code.

Reply:
Right